specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json

```diff
@@ -503,6 +503,48 @@
         }
       }
     },
+    "/providers/Microsoft.Billing/billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/validateDetachPaymentInstrumentEligibility": {
```

Comment:
We have a "payment**Methods**" API today in Billing RP... How about calling this e.g. "validateDetachPaymentMethodEligibility"?

Reply:
Yes updated all the references to paymentinstrument to be payment method